lib/widgets/login.dart

```diff
@@ -400,28 +443,19 @@ class _PasswordLoginPageState extends State<PasswordLoginPage> {
           selectedIcon: const Icon(Icons.visibility_off),
         )));
 
-    return Scaffold(
-      appBar: AppBar(title: Text(zulipLocalizations.loginPageTitle),
-        bottom: _inProgress
-          ? const PreferredSize(preferredSize: Size.fromHeight(4),
-              child: LinearProgressIndicator(minHeight: 4)) // 4 restates default
-          : null),
-      body: SafeArea(
-        minimum: const EdgeInsets.all(8),
-        child: Center(
-          child: ConstrainedBox(
-            constraints: const BoxConstraints(maxWidth: 400),
-            child: Form(
-              // TODO(#110) Try to highlight CZO / Zulip Cloud realms in autofill
-              child: AutofillGroup(
-                child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
-                  usernameField,
-                  const SizedBox(height: 8),
-                  passwordField,
-                  const SizedBox(height: 8),
-                  ElevatedButton(
-                    onPressed: _inProgress ? null : _submit,
-                    child: Text(zulipLocalizations.loginFormSubmitLabel)),
-                ])))))));
+    return Form(
+      // TODO(#110) Try to highlight CZO / Zulip Cloud realms in autofill
+      child: AutofillGroup(
+        child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
+          usernameField,
+          const SizedBox(height: 8),
+          passwordField,
+          const SizedBox(height: 8),
+          ElevatedButton(
+            onPressed: widget.loginPageState._inProgress
+              ? null
+              : _submit,
```

Comment:
nit: this can still fit on one line:
```suggestion
            onPressed: widget.loginPageState._inProgress ? null : _submit,
```

and I think it's slightly easier to read that way, since the `null` is trivial and this brings the `_submit` (which is the main answer) closer to the `onPressed:` (which is the question).